invoice_margin/views/view_account_invoice.xml

```diff
@@ -22,8 +23,9 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
         <field name="inherit_id" ref="account.invoice_form"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='amount_untaxed']/.." position="after">
-                <group name="margin">
+                <group name="margin" string="Margin">
```

Comment:
because of consistency. sale_margin (of Odoo SA) works like that.
https://github.com/odoo/odoo/blob/10.0/addons/sale_margin/views/sale_margin_view.xml#L10